python/GafferTest/AnimationTest.py

```diff
@@ -1469,10 +1469,10 @@ def testModifyKeyReplacesExistingKeyWithInactiveKeys( self ) :
 		k2 = Gaffer.Animation.Key( time = 2 )
 		k3 = Gaffer.Animation.Key( time = 2 )
 
-		curve.addKey( k0, False )
-		curve.addKey( k1, False )
-		curve.addKey( k3, False )
-		curve.addKey( k2, False )
+		curve.addKey( k0, removeActiveClashing = False )
+		curve.addKey( k1, removeActiveClashing = False )
+		curve.addKey( k3, removeActiveClashing = False )
+		curve.addKey( k2, removeActiveClashing = False )
```

Comment:
Seems like we should add some test coverage for the new `inheritInterpolation` parameter as well.

Reply:
will do